Scripts/Mac/CleanupFunctionalTests.sh

```diff
@@ -1,5 +1,8 @@
 . "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
 
+pkill -l git
+pkill -l gvfs
```

Comment:
Will this also take down `gvfs.mount`?

Reply:
We should probably also kill the prjfs-log tool unless that's already handled elsewhere.

Reply:
This will take down gvfs.mount as well as it matches substrings per the man page for pkill. I've validated this locally and confirmed that the PIDs that were killed match both 'gvfs' and 'gvfs.mount' processes that were running.

Thanks for catching `prjfs-log`!